README.md

```diff
@@ -0,0 +1,55 @@
+Hello Microverse
+
+Hello Microverse is a project to demonstrate the understanding of Github, branches, and linter 
+
+
+Built With
+
+* HTML
+
+Live Demo
+
+Getting Started
+
+To get a local copy and running follow these simple steps
+
+*Prerequisites
+  1.a working computer(Desktop or Laptop)
+*Install
+   1.text editor
+   2.git
+*Setup
+    1.create a local folder on your computer
+    2. clone the 'Hello Microverse' project to the local repo
+*Usage
+    1.open the project with a text editor
+
+*Run Tests
+ 1.click Go Live to run the project
+
+ *Depoyment
+   Push the project to remote repository and publish it from the page settings
+
+*Author
+
+GitHub:@Emmy-github-webdev
+Twitter:@ogaemmanueloga
+LinkedIn:@emmanuel-oga-16171584
+
+Contributing
+
+Contributions, issues and feature requests are welcome!
+
+Feel free to check the issues page
+
+Show your support
+
+Give a star if you like this project
+
+Acknowledgements
+
+*Hat tip to anyone whose code was used
+*Inspiration
+
```

Comment:
- [ ] Hello. Your readme file is not well presented. Try to put all the Headings of each section in bold and to do that just add a `#` besides all of them.

Reply:
Thank you. The README.md file has been rearranged